### DISCUSSION TITLE
New Maintainer @w0rmr1d3r

### DISCUSSION THREAD

**Discussion by zricethezav** [zricethezav/gitleaks#623] [General]
Please welcome a new maintainer to the gitleaks team! @w0rmr1d3r. Ramon has helped out with tracking issues and triaging the past couple months when I have been relatively absent. He is the first collaborator to join the team and has been a huge help.

If you're interested in becoming a maintainer I will be providing guidelines on that process in the near future.

Cheers

**Comment by w0rmr1d3r** [comment]
Thank you @zricethezav ! And thank you for your tool!
Let's keep growing!
